src/seedu/addressbook/ui/TextUi.java

```diff
@@ -25,12 +25,6 @@
 
     private static final String DIVIDER = "===================================================";
```

Comment:
Divider should be the role of formatter now. Try to separate the responsibility of TextUI even more such that TextUI class don't need to handle any form of formatting.